models/UnLocationLocation.yaml

```diff
@@ -0,0 +1,35 @@
+name: UnLocationLocation
+type: object
+x-stoplight: 
+  id: b85ad17f907d8
+description: "\t\nAn interface used to express a location using a `UnLocationCode`."
+properties:
+  locationName:
+    type: string
+    format: int64
+    description: The name of the location.
+    pattern: ^\S+(\s+\S+)*$
+    example: Port of Amsterdam
+    maxLength: 100
+  locationType:
+    type: string
+    enum:
+      - UNLO
```

Comment:
Talking about the enum --> string discussions we are having - should we drop the enum here?

Reply:
In this particular case, I believe it's okay to have an enum.